content/language/python/deploy.md

```diff
@@ -1,7 +1,138 @@
 ---
-title: Deploy your app
-keywords: deploy, ACI, ECS, Python, local, development
-description: Learn how to deploy your application
+title: Test your deployment
+keywords: deploy, kubernetes, python
+description: Learn how to develop locally using Kubernetes
 ---
 
-{{< include "deploy.md" >}}
+## Prerequisites
+
+- Complete all the previous sections of this guide, starting with [Containerize a Python application](containerize.md).
+- [Turn on Kubernetes](/desktop/kubernetes/#turn-on-kubernetes) in Docker Desktop.
+
+## Overview
+
+In this section, you'll learn how to use Docker Desktop to deploy your application to a fully-featured Kubernetes environment on your development machine. This allows you to test and debug your workloads on Kubernetes locally before deploying.
+
+## Create a Kubernetes YAML file
+
+In your `python-docker-dev` directory, create a file name `docker-python-kubernetes.yaml`. Open the file in an IDE or text editor and add the following contents.
+Replace `DOCKER_USERNAME/REPO_NAME` with your Docker username and the name of the repository that you created in [Configure CI/CD for your Python application](configure-ci-cd.md).
+
+```yaml
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: docker-python-demo
+  namespace: default
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      service: flask
+  template:
+    metadata:
+      labels:
+        service: flask
+    spec:
+      containers:
+       - name: flask-service
+         image: DOCKER_USERNAME/REPO_NAME
+         imagePullPolicy: Always
+         env:
+          - name: POSTGRES_PASSWORD
+            value: mysecretpassword
+---
+apiVersion: v1
+kind: Service
+metadata:
+  name: service-entrypoint
+  namespace: default
+spec:
+  type: NodePort
+  selector:
+    service: flask
+  ports:
+  - port: 5000
+    targetPort: 5000
+    nodePort: 30001
+```
+
+In this Kubernetes YAML file, there are two objects, separated by the `---`:
+
+ - A Deployment, describing a scalable group of identical pods. In this case,
+   you'll get just one replica, or copy of your pod, and that pod (which is
```

Comment:
I left a comment in the other PR that this could maybe be broken up in to multiple sentences but definitely not blocking!